### PR TITLE
add costomize port in camera

### PR DESCRIPTION
**Pull request by hdy8200**
**What this PR does / why we need it**:
add custom http_port and rtsp_port in camera. when the default port is not used, other ports can be set to connect the camera
**Will this PR make the community happier**? 
yes

**How is this PR tested**
- [ ] unit test
- [x] e2e test
- [ ] other (please specify)

### CODE REVIEW

**Comment by leepala** [comment]
@hdy8200  We need to support these environment variables to be empty to be compatible with older versions.

File: examples/rtspDeviceShifu/camera.py [hunk 1]
```diff
@@ -195,5 +197,12 @@ def move_camera(direction=None):
 
 
 if __name__ == "__main__":
-    video_getter = VideoGet(ip, CAMERA_USERNAME, CAMERA_PASSWORD).start()
+    # Customize port
+    if http_port:
+        http_port = ':' + http_port
+
+    if rtsp_port:
+        rtsp_port = ':' + rtsp_port
```
Comment (tomqin93) [inline examples/rtspDeviceShifu/camera.py:205]: is there a fallback value for these?

**Review by tomqin93 — COMMENTED** [review]
@hdy8200 added 1 comment, please take a look

**Review by tomqin93 — APPROVED** [review]
LGTM

**Pull request merged**